pkgs/applications/window-managers/wmderland/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, libX11, xorgproto }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation {
+  pname = "wmderland";
+  version = "2020-07-17";
```

Comment:
Is there any reasons why not using latest release 1.0.4?

Reply:
It seems that v1.0.4 is so behind from the latest commit and some major features already implemented (e.g. varying split ratio) is missing.

Reply:
Well, add `unstable-` prefix to version at least.

Reply:
Thnak you. I've added it.